treeherder/config/settings.py

```diff
@@ -453,10 +473,17 @@
 BUGFILER_API_URL = env("BUGZILLA_API_URL", default=BZ_API_URL)
 BUGFILER_API_KEY = env("BUGZILLA_API_KEY", default=None)
 
+# For intermittents commenter
+COMMENTER_API_KEY = env("COMMENTER_API_KEY", default=None)
+
 # Log Parsing
 PARSER_MAX_STEP_ERROR_LINES = 100
 FAILURE_LINES_CUTOFF = 35
 
+# Auth0 setup
```

Comment:
This change appears to be from a rebasing issue

Reply:
oops :)